src/avatar.scss

```diff
@@ -0,0 +1,153 @@
+@import "./mixins";
+
+/*!
+.fd-avatar+((--circle), (--thumbnail), (--xxs|--xs|--s|--m|--l|--xl|--xxl))
+*/
+$block: #{$fd-namespace}-avatar;
+
+.#{$block} {
+  // LOCAL VARS (set all themeable properties, always include !default)
+  $fd-const-ratio: 1rem;
+  $fd-avatar-sizes: (
+    xs: $fd-const-ratio * 2,
+    s: $fd-const-ratio * 3,
+    m: $fd-const-ratio * 4,
+    l: $fd-const-ratio * 5,
+    xl: $fd-const-ratio * 7,
+  ) !default;
+  $fd-avatar-border-radius: $fd-border-radius !default;
+  $fd-avatar-border-fade: rgba(0, 0, 0, 0.5); // Should be fade(var(--sapContent_ShadowColor), 10);
+  $fd-avatar-color: var(--sapContent_ContrastTextColor) !default;
+  $fd-avatar-background-color: var(--sapAccentColor6) !default;
+
+  // BLOCK BASE *******************************************
+  @at-root {
+    [class*="#{$block}"] {
+      @include fd-reset();
+
+      color: $fd-avatar-color;
+      background-color: $fd-avatar-background-color;
+      display: inline-block;
+      position: relative;
+      vertical-align: middle;
+      text-align: center;
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-position: 50%;
+      border-radius: $fd-avatar-border-radius;
+    }
+  }
+  // BLOCK MODIFIERS ************
+  // sizes
+  @each $size, $value in $fd-avatar-sizes {
+    &.#{$block}--#{$size} {
+      @include fd-reset-spacing();
+
+      width: $value;
+      height: $value;
+      max-width: $value;
+      max-height: $value;
+      min-width: $value;
+      min-height: $value;
+      font-weight: 400;
+
+      @if $size == xs or $size == s or $size == m {
+        .#{$block}__zoom-icon {
+          width: 1.125rem;
+          height: 1.125rem;
+          bottom: -0.125rem;
+          right: -0.125rem;
+
+          &::before {
+            font-size: 0.75rem;
+          }
+        }
+      }
+
+      @if $size == xs or $size == s {
+        line-height: 1.33333;
```

Comment:
Where is this value coming from? The guidelines recommend 1.4 if we need to set line-height, o.w. it's either 1 or normal.

Reply:
I cannot remember, removing